theme/templates/base.html

```diff
@@ -65,5 +65,17 @@
       {% include "footer.html" %}
       {% endblock %}
     </div>
+
+    <script>
+      const navToggle = document.getElementById('navbar-hamburger-button');
+
+      navToggle.addEventListener('change', () => {
+        if (navToggle.checked) {
+          document.body.classList.add('scroll-lock');
+        } else {
+          document.body.classList.remove('scroll-lock');
+        }
+      });
```

Comment:
`#navbar-hamburger-button` is [a hidden element](https://github.com/mixxxdj/website/blob/9521887c36e3292a040825f5bceaa01015aebe5e/theme/static/css/custom.css#L384) used to persist the expected state of the menu AFAIU, so that `scroll-lock` won't have any effect.